source/winVersion.py

```diff
@@ -217,4 +217,4 @@ def isFullScreenMagnificationAvailable() -> bool:
 	WOW64 applications such as NVDA. For our usages, support has been added since Windows 8, relying on our
 	testing our specific usage of the API with each Windows version since Windows 8
 	"""
-	return getWinVer() >= WIN8
+	return True

```

Comment:
i think it makes sense to deprecate this function to match the strategy used in #15644.
That way usage in NVDA core is discouraged and removed in favour of simpler code.

Reply:
i.e `ScreenCurtainProvider.canStart` can also be simplified to `return True`